ecmabot.js

```diff
@@ -249,6 +249,12 @@ JSBot.prototype.caniuse = function(context, text) {
 	}
 };
 
-var profile = require("./ecmabot-profile.js");
+var profile;
+if (process.env.ECMABOT_PROFILE) {
+    profile = require(path.join(process.cwd(), process.env.ECMABOT_PROFILE));
```

Comment:
i wonder if the thing setting the env var should include `process.cwd()`, rather than causing trouble for someone passing a profile in a totally different path?